chapters/annotations.tex

```diff
@@ -68,27 +73,190 @@ \section{Annotations for Documentation}\doublelabel{annotations-for-documentatio
 \begin{lstlisting}[language=modelica]
 <a href="Modelica://MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
-\begin{lstlisting}[language=grammar]
-preferred-view-annotation:
-   annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+
+\subsection{Annotations for Figures\doublelabel{annotations-for-figures}}
```

Comment:
The labels are normally after the subsection. It could be that they should be inside the sections, but then we should redo everything (in a separate PR).

Reply:
Fixed.  Resolving.